package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "blockparty",
-  "version": "1.0.0",
-  "description": "",
+  "name": "@noblocknoparty/blockparty-contracts",
+  "version": "0.9.0",
+  "description": "BlockParty smart contracts",
   "directories": {
     "doc": "doc",
     "test": "test"
   },
   "scripts": {
+    "prepublishOnly": "truffle compile && truffle networks --clean",
```

Comment:
What about migrate?

Reply:
I don't think you need to run migration as all we need is the json artifact, not necessarily the deployed contract addresses. In fact `--clean` option get rid of addresses.

Reply:
Ok, so we will pass the addresses to the library user in some other way.